SU2_CFD/src/driver_structure.cpp

```diff
@@ -1308,15 +1308,8 @@ void CDriver::Numerics_Preprocessing(CNumerics ****numerics_container,
             default : cout << "Centered scheme not implemented." << endl; exit(EXIT_FAILURE); break;
           }
           
-          if (!config->GetLowFidelitySim()) {
```

Comment:
Does this mean the low fidelity sim. is removed or is this just an improvment ? I found it was a useful feature (although it was not really working properly in the past).